packages/core/src/transport/httpRequest.spec.ts

```diff
@@ -62,44 +68,19 @@ describe('httpRequest', () => {
     expect(navigator.sendBeacon).toHaveBeenCalled()
     expect(server.requests.length).toEqual(1)
   })
-})
-
-describe('httpRequest parameters', () => {
-  const BATCH_BYTES_LIMIT = 100
-  let server: sinon.SinonFakeServer
-  let sendBeaconSpy: jasmine.Spy<(url: string, data?: BodyInit | null) => boolean>
-  beforeEach(() => {
-    server = sinon.fakeServer.create()
-    sendBeaconSpy = jasmine.createSpy()
-    navigator.sendBeacon = sendBeaconSpy
-  })
-
-  afterEach(() => {
-    server.restore()
-  })
 
-  it('should add batch_time', () => {
-    const request = new HttpRequest('https://my.website', BATCH_BYTES_LIMIT, true)
+  it('should have a unique request id', () => {
```

Comment:
since this one tests the integration with endpointBuilder, it could be interesting to separate it from the HttpRequest unit to make it clearer

Reply:
yes, true